packages/secure_storage/amplify_secure_storage_dart/pubspec.yaml

```
@@ -48,3 +48,9 @@ flutter:
   assets:
     - lib/src/worker/workers.min.js
     - lib/src/worker/workers.min.js.map
+  plugin:
+    platforms:
+      linux:
+        default_package: amplify_secure_storage_dart
```

`default_package` would typically refer to another package. This references itself. Is that something flutter docs suggest doing?

What I am seeing in the docs is to use `dartPluginClass` 

```suggestion
        dartPluginClass: AmplifySecureStorageDart
```

You will also have to add a `registerWith()` method to AmplifySecureStorageDart I believe. This is based on https://docs.flutter.dev/packages-and-plugins/developing-packages#dart-only-platform-implementations